packages/@aws-cdk/aws-servicediscovery/lib/non-ip-instance.ts

```diff
@@ -0,0 +1,58 @@
+import cdk = require('@aws-cdk/cdk');
+import { BaseInstanceProps, InstanceBase } from './instance';
+import { NamespaceType } from './namespace';
+import { IService } from './service';
+import { CfnInstance } from './servicediscovery.generated';
+
+// tslint:disable-next-line:no-empty-interface
+export interface NonIpInstanceBaseProps extends BaseInstanceProps {
```

Comment:
This isn't used anywhere, except for declaring `NonIpInstanceProps` just below. I feel like this is unnecessary.

Reply:
True, but I thought that it would be good to keep the naming consistent and makes it easier to extend later if neccessary.